step/util.go

```diff
@@ -0,0 +1,36 @@
+package step
+
+type InfoLogger interface {
+	Infof(format string, v ...interface{})
+}
+
+// LogWriter can be used in place of an io.Writer where we want to redirect the output to a logger in real time
+type LogWriter struct {
```

Comment:
That would only output the logs after the build has finished which could be a while. This is both bad UX and I believe there is a timeout for steps that might kill it this way.

If you mean the actual os stdout, that would not go through the step logger so it will not be the same structure making it probably look out of place, or hard to parse.